app/views/admin/report_notes/_report_note.html.haml

```diff
@@ -1,18 +1,24 @@
-.report-notes__item
+.report-notes__item{ id: dom_id(report_note) }
   = image_tag report_note.account.avatar.url, class: 'report-notes__item__avatar'
 
   .report-notes__item__header
     %span.username
       = link_to report_note.account.username, admin_account_path(report_note.account_id)
-    %time.relative-formatted{ datetime: report_note.created_at.iso8601 }
-      = l report_note.created_at.to_date
+    %a.timestamp{ href: "##{dom_id(report_note)}" }
+      %time.relative-formatted{ datetime: report_note.created_at.iso8601 }
+        = l report_note.created_at.to_date
 
   .report-notes__item__content
-    = linkify(report_note.content)
+    - if report_note.is_a?(InstanceNote)
+      = report_note.formatted_content
+    - else
+      = linkify(report_note.content)
```

Comment:
We probably want to make a decision if the notes functionality supports markdown, and if so, apply it everywhere all at once.